eos/form-factors/baryonic_TEST.cc

```diff
@@ -79,6 +80,96 @@ class BFvD2014FormFactorsTest :
         }
 } bfvd2014_form_factors_test;
 
+class DKMR2017FormFactorsTest :
```

Comment:
Please merge the commit introducing the test cases with the commit introducing the new code. It's a good choice to always keep both together.